src/components/forms/SectionSelector.tsx

```diff
@@ -32,50 +27,30 @@ interface FormSelectWrapperProps {
   selected: boolean;
 }
 
-const FormSelectWrapper = styled(LocalizedLink)<FormSelectWrapperProps>`
-  display: block;
-  color: inherit;
-  padding: 1em;
-  text-align: center;
-  min-height: 8em;
-  &:hover,
-  &:active {
-    text-decoration: none;
-    color: inherit;
-  }
-
-  svg {
-    max-height: 6em;
-  }
-
-  background-color: ${props => (props.selected ? props.theme.helFog : 'unset')};
-  font-size: 0.8em;
-  line-height: 1;
-  ${responsive.sm`
-    font-size: 1em;
-    font-weight: ${(props: FormSelectWrapperProps) => (props.selected ? '600' : '400')};
-  `}
-`;
-
 const SectionSelector = ({ name, selected, types, sizes }: Props) => (
-  <ButtonWrapper>
+  <div className="app-Form__section-selector">
```

Comment:
For the sake of consistency, how about `app-SectionSelector`?

Reply:
Sure.